pkg/apis/options/load_test.go

```diff
@@ -541,3 +542,63 @@ injectResponseHeaders:
 		}))
 	})
 })
+
+func TestLoadYAML(t *testing.T) {
```

Comment:
Can you please rewrite this test suite in the same style as the other tests using Ginkgo, either with or without a table, but I'd like to try and keep the tests consistent please

Reply:
done + removed cases that were already covered in the ginkgo tables